R/repr_matrix_df.r

```diff
@@ -19,12 +19,21 @@ ellip.d <- '\u22F1'
 
 ellipses <- c(ellip.h, ellip.v, ellip.d)
 
+get.limit.index <- function(obj_dim, limit) {
+	stopifnot(obj_dim > limit)  # otherwise this function should not have been run
+	left_or_top <- seq_len(ceiling(limit / 2))
+	right_or_bottom <- seq.int(obj_dim - floor(limit / 2) + 1L, obj_dim)
+	return(list(begin=left_or_top, end=right_or_bottom)) 
+}
+
 ellip.limit.vec <- function(v, num, ellip) {
 	stopifnot(num >= 2L)
 	
-	left  <- seq_len(ceiling(num / 2))
-	right <- seq.int(length(v) - floor(num / 2) + 1L, length(v))
-	
+	lim_index <- get.limit.index(length(v), num)
+	left <- lim_index$begin
+	right <- lim_index$end
+	rm(lim_index)
```

Comment:
i’d just call it `lims` and use `lims$begin` and `lims$end` later. saves 3 lines and is no less clear